src/main/java/hudson/plugins/sloccount/SloccountAreaRenderer.java

```diff
@@ -10,6 +11,7 @@
  * links. This renderer does not render tooltips, these need to be defined in
  * sub-classes.
  */
+@SuppressFBWarnings(value="EQ_DOESNT_OVERRIDE_EQUALS", justification="Equals method is not needed.")
```

Comment:
Not sure about it, seems to be a valid bug

Reply:
Then add a TODO comment which says "should be fixed". And probably link comment to a Jenkins ticket